mmv1/templates/terraform/examples/cloudbuild_trigger_service_account.tf.erb

```diff
@@ -0,0 +1,27 @@
+resource "google_cloudbuild_trigger" "<%= ctx[:primary_resource_id] %>" {
+  trigger_template {
+    branch_name = "master"
+    repo_name   = "my-repo"
+  }
+
+  service_account = "${google_service_account.cloudbuild_service_account.id}"
```

Comment:
```suggestion
  service_account = google_service_account.cloudbuild_service_account.id
```

Mostly a nitpick- your syntax is required to interpolate inn the middle of a string, but there's a nicer one that's recommended in `0.12+`.

Reply:
Updated.